pkg/cmd/create/create_terraform.go

```diff
@@ -127,6 +128,7 @@ func (options *CreateTerraformOptions) addFlags(cmd *cobra.Command, addSharedFla
 	cmd.Flags().StringVarP(&options.Flags.ClusterName, optionClusterName, "", "", "The name of a single cluster to create - cannot be used in conjunction with --"+optionCluster)
 	cmd.Flags().StringVarP(&options.Flags.CloudProvider, optionCloudProvider, "", "", "The cloud provider (currently gke only) - cannot be used in conjunction with --"+optionCluster)
 	cmd.Flags().BoolVarP(&options.Flags.SkipTerraformApply, "skip-terraform-apply", "", false, "Skip applying the generated Terraform plans")
+	cmd.Flags().BoolVarP(&options.Flags.SkipJxInstall, "skip-jx-install", "", false, "Skip installing Jenkins X into the generated cluster")
```

Comment:
we should make this consistent (name, help text) with the gke command https://jenkins-x.io/commands/jx_create_cluster_gke/